app/commands/v2/put_content.rb

```diff
@@ -221,6 +222,14 @@ def update_content_item(content_item)
         content_item.save!
       end
 
+      def update_last_edited_at_if_needed(content_item, last_edited_at = nil)
+        if last_edited_at.nil? && %w(major minor).include?(payload[:update_type])
+          last_edited_at = Time.zone.now
```

Comment:
Ah, so sounds like there is a case where if you explicitly set the last_edited_at, then it should use that (regardless of the update_type) Is that correct @elliotcm? If so we need a test around that and probably use the original method.